### ISSUE TITLE
Expose the Basic Contact List Information

### ISSUE THREAD

**Issue by dLopreiato** [YUKV/contacts#2]
Write an API that would allow angular to access a user's contact list.
 - User's should not be able to access another user's contact list
 - API must be able to handle both pagination and full dumps
    - Pagination would take parameters "count" and "page"
 - Be prepared to be able to take more fields
 - Be prepared to be able to select which fields to return

**Comment by dLopreiato** [comment]
> Is a human list just the list of people someone has interacted with?

No. At the core of this app, exists a basic contacts list. Just like you'd find in your phone. This is what the human list is.

**Comment by WBittner** [comment]
Do we have the list of fields a human list will contain? This task seems dependent on the schema task, #1
